Shipr-Helm/templates/shipr-inventory-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.inventory.configMapName }}
```

Comment:
labels, annotations? 

Please read more here, if you are not familiar with why annotations are important. 
https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
Done